rest/utilities_testing.go

```diff
@@ -181,8 +181,8 @@ func (rt *RestTester) Bucket() base.Bucket {
 	} else if rt.RestTesterConfig.persistentConfig {
 		// If running in persistent config mode, the database has to be manually created. If the db name is the same as a
 		// past tests db name, a db already exists error could happen if the past tests bucket is still flushing. Prevent this
-		// by setting the group ID as the current test name by default.
-		sc.Bootstrap.ConfigGroupID = fmt.Sprintf("%x", sha256.Sum256([]byte(rt.TB.Name())))
+		// by using a unique group ID for each new rest tester.
+		sc.Bootstrap.ConfigGroupID = fmt.Sprintf("%d-%d-%d", rand.Uint64(), rand.Uint64(), rand.Uint64())
```

Comment:
Why three different Uint64?  If you're looking for a uniqueness guarantee, maybe just use UUIDs?